docs/plotting.md

```diff
@@ -76,13 +92,14 @@ that makes profit spikes.
 
```

Comment:
^ your algorithm

Reply:
sorry i don't understand this?

Reply:
It's written In the sentence above this line "show how you algorithm works". Change "you algorithm" --> "your algorithm" there. Cannot press a "+" button straight on that line so made this comment below.